### ISSUE TITLE
When naming a preset, if you use special char, it prompts strange error message

### ISSUE THREAD

**Issue by vargasjj** [fxsound2/fxsound-app#67]
**Describe the bug**
When you name a preset and happen to typo or use some special characters, the naming fails and I get weird techy errors instead of a common english error like "invalid character" or something like that.

I used forward-slash / on the name by accident and I got a "75: ptutil\VALS\Valsfile.cpp" error and then a second error prompts "91: DfxDspPreset.cpp" error

**Expected behavior**
Either permit special characters on the profile names, or guide user with appropriate character expectations from the name on the error message to notify users to avoid using non-supported characters.

**Screenshots**

![image](https://github.com/fxsound2/fxsound-app/assets/14932891/74b1d1f8-5273-4202-ab92-512a420ae7ba)

![image](https://github.com/fxsound2/fxsound-app/assets/14932891/eae2a259-c51d-4de2-bb6f-182ec6c7b312)

**System information**
 - Windows version: Windows 11 23H2 (22631.2861)
 - Audio device: Regular laptop audio output speakers (Speakers Intel® Smart Sound Technology for MIPI SoundWire® Audio)  

**Additional context**
Nothing more.

Additional Thanks: Thanks for this great app. Best EQ so far on the web for Windows I've come across, and I can't believe it is free. I was happy to donate and post this issue to contribute the project. Please keep the hard work. Greetings from Barranquilla, Colombia.

JJ

**Comment by doolhoofd** [comment]
Thank you for pointing this out. It has been forwarded to @bvijay74 and @jamesarcher9000.

**Comment by bvijay74** [comment]
These special characters are already filtered and not allowed to enter while naming the preset because these reserved characters cannot be used in naming the preset file.
< > : " / \ | ? *

Which special character are you entering which caused this exception? Also, which version of FxSound are you using?

**Comment by vargasjj** [comment]
@bvijay74 thank you for your reply.
Answer to your questions:

1. I used forward-slash. it is this char -> "/"
2. I was using 1.1.16.0 but I am about to update to the latest one just to make sure if the error is still there.

I am not interested on using any special char for the preset naming. I encounter this failure by accident.
I just consider that the error log is not common english for an end user and it is confusing.

**Comment by vargasjj** [comment]
@bvijay74 I just updated to 1.1.20.0 and I confirmed I cannot longer use special characters on the preset naming. it is just not allowing me to enter or type them in the name so there's no way i can use them for the name anymore, s o the filter you mentioned is working OK.

The error was there on 1.1.16.0 but I guess we can archive this bug now as the issue was resolved somewhere between 1.1.16.0 and 1.1.20.0

thank you all!
